front/components/assistant_builder/types.ts

```diff
@@ -76,10 +76,7 @@ export type AssistantBuilderProcessConfiguration = {
 };
 
 // Websearch configuration
-
-export type AssistantBuilderWebsearchConfiguration = {
-  searchResults: 8; // not configurable at the time of writing, placeholder
-};
+export type AssistantBuilderWebsearchConfiguration = Record<string, never>; // no relevant params identified yet
```

Comment:
Shouldn't we get rid of that?

Reply:
You mean remove the AssistantBuilderWebsearchConfig type?

This would mean several adjustments in code for the specific case of the Websearch action, since all actions are currently expected to have a config. For the sake of consitency/idomacy/simplicity, I'd rather leave it and enforce it to be empty (via the Record<string, never>)

(+ it also makes future addition of params easier as a nice to have)

Happy to discuss if needed

Reply:
kk